src/core/harpoon.ml

```diff
@@ -239,28 +239,27 @@ module Tactic = struct
          | Coverage.CovCtx _
            | Coverage.CovGoal (_, _, _) ->
             Misc.not_implemented "CovCtx impossible"
-         | Coverage.CovPatt (cG, p, tau) ->
-            let open Comp in
+         | Coverage.CovPatt (cG, pat, tau) ->
             let refine_ctx ctx = Whnf.cnormCtx (Whnf.normCtx ctx, ms) in
-            let cG = refine_ctx s.context.cG in
+            let cG = Coverage.compgctx_of_gctx cG in
```

Comment:
This is suspicious. Is `cG` in `CovPatt` the context _extension_ listing the computational variables in the pattern? Or is it the complete context, with the extensions?
Anyway, I think I understand why this change was made: `refine_ctx` couldn't possibly be introducing new computational variables, which `comp-split` definitely needs to do.

Reply:
@tsani As far as I checked, it is the complete context, though I'm not 100% sure.